src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -46,13 +46,12 @@ object Homework :
       else true
 
     def and(left: Boolean, right: Boolean): Boolean =
-      if left == right then true
-      else false
+      if left then right
+      else not(right)

```

Comment:
I'd prefer jus `else false`, it's more clear.

Reply:
confused with equivalence somehow...

Reply:
Didn't get your question (if it was a question).

Reply:
this is statement)